.github/workflows/create-beta-pull-request.yml

```diff
@@ -1,66 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-# This action will automatically create a pull request against dev if the pushed branch
-# has a branch path spec like kiota/beta/pipelinebuild/*. Configure this action by updating the
-# environment variable values[0].
-
-name: "create beta pull request"
-
-# Controls when the action will run. Triggers the workflow on push
-# events but only for branches with the following branch spec: "kiota/beta/pipelinebuild/*"
-on:
-  push:
-    branches:
-      - "kiota/beta/pipelinebuild/*"
-    paths:
-      - 'src/Generated/**'
-
-defaults:
-  run:
-    shell: bash
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
-jobs:
-  bump-sdk-version:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: Git config user
-      run: |
-        git config --global user.email "GraphTooling@service.microsoft.com"
-        git config --global user.name "Microsoft Graph DevX Tooling"
-
-    - name: Run increment script
-      run: php scripts/BumpPreviewSdkVersion.php
-
-    - name: Commit and push changes if any
-      run: if git commit -am "Bump SDK version"; then git push origin $GITHUB_REF; fi
```

Comment:
We'll still need to bump the SDK version. We can rename this workflow & only deprecate the PR job?

Reply:
Working on it in separate PR